lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2483,49 +2484,45 @@ def quiver(self, *args,
                length=1, arrow_length_ratio=.3, pivot='tail', normalize=False,
                **kwargs):
         """
-        Plot a 3D field of arrows.
-
-        call signatures::
-
-            quiver(X, Y, Z, U, V, W, **kwargs)
-
-        Arguments:
+        ax.quiver(X, Y, Z, U, V, W, /, length=1, arrow_length_ratio=.3, pivot='tail', normalize=False, **kwargs)
 
-            *X*, *Y*, *Z*:
-                The x, y and z coordinates of the arrow locations (default is
-                tail of arrow; see *pivot* kwarg)
-
-            *U*, *V*, *W*:
-                The x, y and z components of the arrow vectors
+        Plot a 3D field of arrows.
 
         The arguments could be array-like or scalars, so long as they
```

Comment:
probably should add `The arguments *X*, *Y*, *Z*, *U*, *V*, *W*` because the context above is deleted.